pkg/minishift/provisioner/minishift_provisioner.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provisioner
```

Comment:
+1 for the new package structure. I think we might be able to drop provisioner in the file name. Coming from Java, _minishift_provisioner_ sounds good, but I think in the Go world one preders shorter  names.